app/src/main/java/com/example/lawuna/HomeActivity.java

```diff
@@ -81,28 +81,15 @@ public void onClick(View view) {
         });
         DrawerLayout drawer = findViewById(R.id.drawer_layout);
         NavigationView navigationView = findViewById(R.id.nav_view);
+        View headerView = navigationView.getHeaderView(0);
+        TextView navUsername = (TextView) headerView.findViewById(R.id.user_name);
+//        Show the current user
+        navUsername.setText("Active User");
 
         Intent intent = getIntent();
         phone_number = intent.getStringExtra("phone_number");
         Log.d(TAG, "onCreate: "+phone_number);
 
-
-//        mViewPager = (ViewPager) findViewById(R.id.frag_view);
-////        setup Pager
-//        setupViewPager(mViewPager);
-//        Access the fragment
-//        Fragment fragment = new SlideshowFragment();
-////        Transition
-//        FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
-////        Replace whatever is in the Legal fragment view with this fragment and transition to the back stack
-//        transaction.add(R.id.child_fragment, fragment);
-////        transaction.setTransition(new LegalFragment().getId());
-//
-////        transaction.addToBackStack(null);
-////        Commit transition
-//        transaction.commit();
-
-
```

Comment:
Its good that your removing the unused code from the files. Please do that for the whole code base, thanks.

Reply:
Alright then, let me work on that.